src/main/java/org/opentripplanner/routing/trippattern/TripTimes.java

```diff
@@ -49,6 +48,8 @@
      */
     private final String[] headsigns;
 
+    private final String[][] vias;
```

Comment:
This class is one of the most important classes in OTP, and almost all fields even private ones have JavaDoc. Throughout OTP the style is different, but the "rule" is to comment on the `get` accessor, not field and setter. But, taken into account the importance of this class, we would like it to keep the existing style; Hence comments on both field an getter for now. See the comments on the accessor for what to include in the JavaDoc.

Reply:
JavaDoc added.